### DISCUSSION TITLE
Font used in Stellarium Web

### DISCUSSION THREAD

**Discussion by EGPlanetarium** [Stellarium/stellarium#3856] [Q&A]
Hi, I am Gianluca, 
What is the font used for constellations in Stellarium Web? and how can I change it for stellarium windows 11 64 bit version ?
Thank you.
Gianluca

**Comment by gzotti** [comment]
Another question that should better be asked in the Stellarium Web forum...

**Reply by EGPlanetarium** [reply]
Sorry i am not able to ask the question in the forum

**Reply by gzotti** [reply]
You could raise an issue then. The desktop developers here cannot answer your question.

**Comment by lulunac27a** [comment]
The font used for the whole [Stellarium Web](https://stellarium-web.org/) interface text, including the text used for constellations, is Roboto.
